packages/config/.gitignore

```diff
@@ -0,0 +1,6 @@
+bin
+lib
+!src/bin
+!src/bin/*
```

Comment:
Do we really need both entries the folder and the wildcard here?

Reply:
Yes. https://stackoverflow.com/questions/5533050/gitignore-exclude-folder-but-include-specific-subfolder

TLDR
```
You must white-list folders first, before being able to white-list files within a given folder.
```